crates/iota-indexer/src/store/pg_indexer_store.rs

```diff
@@ -262,7 +272,9 @@ impl PgIndexerStore {
                 .first::<(i64, Option<i64>)>(conn)
                 .map(|(min, max)| (min as u64, max.map(|v| v as u64)))
         })
-        .context("Failed reading checkpoint range from PostgresDB")
+        .context(
+            format!("Failed reading checkpoint range from PostgresDB for epoch {epoch}").as_str(),
```

Comment:
Lowercase for error messages

```suggestion
            format!("failed reading checkpoint range from PostgresDB for epoch {epoch}").as_str(),
```